code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -233,6 +233,9 @@ Applications: 8 servants, 3 caches, and 100 CV
 	if(one_per_tile && (locate(prevent_path) in T))
 		to_chat(invoker, "<span class='warning'>You can only place one of this object on each tile!</span>")
 		return FALSE
+	if(locate(/obj/structure/destructible/clockwork/trap) in T)
```

Comment:
this will trigger on literally every object-creating scripture, even if that scripture creates an item.

Reply:
also you didn't... prevent placing the sigil, THEN the trap, so doesn't actually fix the problem

Reply:
yeah that occurred to me a bit ago, I'll fix it in a few once I finish some rl things